python/hail/docs/tutorials/hail-overview.ipynb

```diff
@@ -347,12 +313,20 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": true,
     "scrolled": true
    },
    "outputs": [],
    "source": [
-    "pprint(ds.col_schema)"
+    "print(ds.col.dtype)"
```

Comment:
Do you need both of these cells?

Reply:
yeah, seeing the pretty vs normal str is useful I think